package.json

```diff
@@ -2383,15 +2383,45 @@
 						"default": null,
 						"items": {
 							"type": "object",
-							"required": [
-								"prefix",
-								"url"
+							"oneOf": [
```

Comment:
This produces:


https://user-images.githubusercontent.com/29582865/194799758-e95bd06b-6cac-4f2e-b7de-8d5824d77b78.mp4